docs/guide/contributing.rst

```diff
@@ -1,73 +1,203 @@
+1.  Open a terminal.
 
-If you want to add a new set of topics, consider where they would fit best into the rest of the guide. If it's not obvious where the new topics should go, ask in Slack, or file an issue.
+    * In macOS: open the **Terminal** app.
+    * In Windows: from the Start Menu, open **Git Bash**.
 
-The main focus is general principles and best practices, and we'd like to keep it that way. This means no bikeshedding -- arguing over minor points at the expense of major principles of clarity and communication. And do avoid preferential editing -- offering only your personal preferences for dealing with a particular writing situation. If you recommend a particular word choice, for example, explain why it matters to the larger world of the audience you want to address.
+2.  Go to a directory for storing the cloned repository. 
 
-What we expect from you, and what you can expect from us
---------------------------------------------------------
+3.  Type ``git clone``, followed by a space,
+    and then paste the project URL:
 
-We encourage contributions from experienced folks and from newcomers, whether you're new to documentation, to software, to GitHub, or to Write the Docs. We expect you to respect the effort that's gone into whatever the current state of the guide might be, and we'll treat your contributions with respect in turn.
+       ::
 
-Keep your tone friendly and encouraging. It's a good way to write docs in general, so practice with docs about docs!
+          git clone https://github.com/myname/www.git
 
-Link liberally to resources that you find useful, but make sure to provide appropriate attribution. And make sure to add something about how or why you find them useful.
+    git clone copies files from GitHub to a folder named ``www`` on your computer.
 
-For anyone new to GitHub and Git
---------------------------------
+4.  Go to the ``www`` directory:
 
-If you've never worked with Git and GitHub before, this resource is great for getting started: `How to Contribute to an Open Source Project on GitHub <https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github>`_.
+       ::
 
-To suggest small changes, you can `submit issues <https://github.com/writethedocs/www/issues>`_ via the GitHub interface. 
+          cd www
 
-For large changes, you'll probably want to learn how to fork and branch, how to work locally (although you can do most things in the GitHub interface in your branch), and how to keep your fork in sync with the main repository.
+5.  Create a new branch:
 
-If you really don't want to deal with GitHub at all, we'll take attachments or inline text emailed to guide@writethedocs.org. But we'd like to encourage you to learn some of the basic tools that we use.
+       ::
 
-Markup
-------
+          git branch branch-name
```

Comment:
If we're not showing command output, we could just use ``git command`` instead of the blog syntax, but it's not worth blocking this one.